nym-vpn-core/nym-vpnd/src/win_service/service.rs

```diff
@@ -0,0 +1,172 @@
+use std::env;
+use std::ffi::OsString;
+use std::time::Duration;
+
+use nym_task::TaskManager;
+use tokio::sync::broadcast;
+use tracing::info;
+use windows_service::service::{
+    ServiceControl, ServiceControlAccept, ServiceDependency, ServiceErrorControl, ServiceExitCode,
+    ServiceInfo, ServiceStartType, ServiceState, ServiceStatus, ServiceType,
+};
+use windows_service::service_control_handler::{self, ServiceControlHandlerResult};
+use windows_service::service_dispatcher;
+
+use crate::cli::CliArgs;
+use crate::command_interface::start_command_interface;
+// use crate::service::start_vpn_service;
+
+use super::install;
+
+windows_service::define_windows_service!(ffi_service_main, my_service_main);
+
+pub(crate) static SERVICE_NAME: &str = "nym-vpnd";
+pub(crate) static SERVICE_DISPLAY_NAME: &str = "NymVPN Service";
+
+pub(crate) static SERVICE_DESCRIPTION: &str =
+    "A service that creates and runs tunnels to the Nym network";
+static SERVICE_TYPE: ServiceType = ServiceType::OWN_PROCESS;
+
+fn my_service_main(arguments: Vec<OsString>) {
```

Comment:
This looks like an example type function name. Maybe something like `nym_service_main` instead?

Reply:
Yeah this is leftover from example code, I'll change it!